test/support/test_helpers.ex

```diff
@@ -1,15 +1,33 @@
 defmodule RemoteRetro.TestHelpers do
   use Wallaby.DSL
-  alias RemoteRetro.Repo
+  alias RemoteRetro.{Repo, User}
 
   def persist_idea_for_retro(context) do
-    %{idea: idea, retro: retro} = context
-    idea = Map.put(idea, :retro_id, retro.id)
+    %{idea: idea, retro: retro, user: user} = context
+    user =
+      if Map.has_key?(user, :id) do
```

Comment:
Rather than use the id here, and falling back to email in cases where there isn't an id, could we just augment the mock user with an id?